### ISSUE TITLE
[OSIO] Usability issues

### ISSUE THREAD

**Issue by hrishin** [openshiftio/openshift.io#1914]
I'm using springboot quickstart application and added following dependency in the `pom.xml`
```
   <dependency>
        <groupId>io.fabric8</groupId>
        <artifactId>kubernetes-client</artifactId>
        <version>3.1.8</version>
    </dependency>
```

I'm facing following issues while developing an application

- [che] Import package: It's really not easy to add import statement with alt + ctrl popup. Import option gets hide somewhere bellow of in between. It's slowing down my velocity.
![import](https://user-images.githubusercontent.com/28615674/35088618-59fe815e-fc5a-11e7-8ba1-18e2036d2f3f.png)

- [build] Compilation errors: It's not able to find package during compilation.

```
ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /projects/dashboard/src/main/java/io/openshift/booster/service/GreetingEndpoint.java:[25,35] package io.fabric8.openshift.client does not exist
[ERROR] /projects/dashboard/src/main/java/io/openshift/booster/service/GreetingEndpoint.java:[40,13] cannot find symbol
  symbol:   class OpenShiftClient
```
![compile_error](https://user-images.githubusercontent.com/28615674/35088259-405b7410-fc59-11e7-9ad4-c006df35094c.png)

**Comment by eivantsov** [comment]
@hrishin did you try to right click the project > Maven - > reimport?

**Comment by slemeur** [comment]
@hrishin : The first issue that you are mentioning is that the completion proposals (and quick actions) are not getting sorted in an efficient way. Correct?

**Comment by hrishin** [comment]
@eivantsov I tried sorry to say but didn't worked.

**Comment by eivantsov** [comment]
@hrishin do you see this lib in dependencies? There's a folder in the bottom of the project.

**Comment by hrishin** [comment]
@slemeur Yes, it would be nice if mostly used option appears on the top and used options bellow.

**Comment by eivantsov** [comment]
![image](https://user-images.githubusercontent.com/5337267/35093418-03608496-fc4b-11e7-87c1-be6ce20dad13.png)

Cannot reproduce locally with Che 6. 

I assume that the library download failed and thus it has never been added to classpath.

**Comment by hrishin** [comment]
@eivantsov ahaan my bad, its dependency issue. Got it fixed now.

**Comment by hrishin** [comment]
@eivantsov just wondering why it's not showing warning in IDE

**Comment by eivantsov** [comment]
@hrishin there should be a warning in pom.xml

**Comment by hrishin** [comment]
@eivantsov Yes I can see vulnerability warning.
![dependency_issue](https://user-images.githubusercontent.com/28615674/35095598-018d0724-fc6f-11e7-938f-f901997131c6.png)

**Comment by eivantsov** [comment]
@hrishin I mean if the lib was never downloaded, the warning should say smth about it, not merely LS warning

**Comment by ibuziuk** [comment]
@hrishin am I correct the issue can be closed now since it was basically a dependency problem ?

[1] https://github.com/openshiftio/openshift.io/issues/1914#issuecomment-358614550

**Comment by hrishin** [comment]
Yes @ibuziuk
